src/Timeline.js

```diff
@@ -73,6 +81,12 @@ export default class Timeline extends Axis {
 
       const pixelDomain = domain.map(this._d3Scale);
 
+      if (this._buttonBehaviorCurrent === "buttons") {
+        const desv = 0.5 * (this._width / this._availableTicks.length - this._handleSize);
+        pixelDomain[0] -= desv;
+        pixelDomain[1] += desv;
+      }
```

Comment:
this `desv` and single `+/- 0.1` logic is repeated 4 times, can you:
1. create a helper method that accepts the current domain as an argument and returns a new modified domain
2. rename the `desv` variable... what does that stand for?

Reply:
To be clear, I am proposing creating a function outside of the Class that modifies a given domain and contains the following code:

```js
if (selection[0] === selection[1]) {
  selection[0] -= 0.1;
  selection[1] += 0.1;
}

if (this._buttonBehaviorCurrent === "buttons") {
  const brushWidth = 0.5 * (this._width / this._availableTicks.length - this._handleSize);
  selection[0] -= brushWidth;
  selection[1] += brushWidth;
}
```

Reply:
done!